### DISCUSSION TITLE
Complex Menu component

### DISCUSSION THREAD

**Discussion by phuoccss** [adobe/react-spectrum#6029] [Q&A] [answered]
Is it possible to make Menu that contains a mix of standard menu items, checkbox group and radio group, similar to [Radix Dropdown Menu](https://www.radix-ui.com/primitives/docs/components/dropdown-menu)?

![Screen Shot 2024-03-09 at 12 00 11](https://github.com/adobe/react-spectrum/assets/15166715/946e908d-d2cd-4b13-905c-4bdd566be12a)

**Comment by snowystinger** [comment, marked as answer]
Not yet, but it's something we'd like to eventually support, see https://github.com/adobe/react-spectrum/issues/1621

**Reply by phuoccss** [reply]
thanks for your reply
